### DISCUSSION TITLE
side reserve (green strip between sidewalk/cycleway and main road)

### DISCUSSION THREAD

**Discussion by RubenKelevra** [streetcomplete/StreetComplete#4305] [Ideas]
I'm not sure there's even a tag(ging convention) for this, but I would really like to have this information inside of OSM - somehow.

Having it would allow routers to select "greenish" and much safer routes more easily for bicycles and foot traffic.

Here's how a side reserve (DE:Seitentrennstreifen) looks like:

![Seitentrennstreifen svg](https://user-images.githubusercontent.com/614929/186119612-e4cdd1c5-d045-45f5-929a-0ade6912fc3e.png)
Source: [Link](https://commons.wikimedia.org/wiki/File:Seitentrennstreifen.svg)

---

- Anyone got an idea how to tag this (without creating a dedicated area)?
- Do you guys think it would pass the checklist for new quests?

> ### Checklist
> Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
> - [ ] 🚧 To be added tag is established and has a useful purpose
> - [ ] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
> - [ ] 🐿️ Easily answerable by any pedestrian from the outside but a survey is necessary
> - [ ] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
> - [ ] 🕓 Applies to a reasonable number of map data (Worth the effort)

---

### Similar tags

There's a tag for a [verge](https://wiki.openstreetmap.org/wiki/Key:verge), which is confusing, as this is called a verge in German:

![Bankett svg](https://user-images.githubusercontent.com/614929/186137595-5e4edfa4-525e-410e-a29d-19f3fc7b4cdd.png)

A side reserve (DE:Seitentrennstreifen) on the other hand has solely the purpose of separating the car traffic from pedestrian and cyclist traffic.

### Wikipedia

Wikipedia documents multiple terms for them, I feel like the best fitting would be "curb lawn" or "curb strips":

> Curb lawn: [Kalamazoo, Michigan](https://en.wikipedia.org/wiki/Kalamazoo,_Michigan);[[17]](https://en.wikipedia.org/wiki/Road_verge#cite_note-17) [Elyria, Ohio](https://en.wikipedia.org/wiki/Elyria,_Ohio);[[18]](https://en.wikipedia.org/wiki/Road_verge#cite_note-18) [Miami County, Ohio](https://en.wikipedia.org/wiki/Miami_County,_Ohio);[[19]](https://en.wikipedia.org/wiki/Road_verge#cite_note-19)[[20]](https://en.wikipedia.org/wiki/Road_verge#cite_note-20) [Greenville, South Carolina](https://en.wikipedia.org/wiki/Greenville,_South_Carolina)[[21]](https://en.wikipedia.org/wiki/Road_verge#cite_note-21)
> Curb strip: [New Jersey](https://en.wikipedia.org/wiki/New_Jersey), [New York](https://en.wikipedia.org/wiki/New_York_(state)), [North Carolina](https://en.wikipedia.org/wiki/North_Carolina), [Florida](https://en.wikipedia.org/wiki/Florida), Ohio, Indiana, [Massachusetts](https://en.wikipedia.org/wiki/Massachusetts), [Michigan](https://en.wikipedia.org/wiki/Michigan), [Iowa](https://en.wikipedia.org/wiki/Iowa), [Kansas](https://en.wikipedia.org/wiki/Kansas), [Nebraska](https://en.wikipedia.org/wiki/Nebraska), [Oregon](https://en.wikipedia.org/wiki/Oregon), [Washington](https://en.wikipedia.org/wiki/Washington_(state))[[12]](https://en.wikipedia.org/wiki/Road_verge#cite_note-dictionary-vol6-12)

https://en.wikipedia.org/wiki/Road_verge#Terminology

**Comment by andrewharvey** [comment]
This would be a great new quest to add, and I agree with the routing utility.

> Wikipedia documents multiple terms for them, I feel like the best fitting would be "curb lawn" or "curb strips":

Ideal case would be using the localised term, which would be hard when there is so much variation in the US by state (according to your link) but at least it can be done by language region as a start.

> There's a tag for a [verge](https://wiki.openstreetmap.org/wiki/Key:verge), which is confusing, as this is called a verge in German:

I believe that's the tag you want, but don't see how that's confusing the tag name would be localised as Seitentrennstreifen so German users wouldn't see the work "verge" at all.

**Reply by RubenKelevra** [reply]
Well my point was, that there are seperate words in German for a verge (which is just the greenish end of a road), like on this picture:

![Protected_road_verge_-_geograph org uk_-_1800031](https://user-images.githubusercontent.com/614929/186225471-13c8ac4a-ce3b-47d5-9107-66298145edbd.jpg)


And a grass divider between the sidewalk/cycle way and the main road - like in this picture:

![Oak_Park_Boulevard](https://user-images.githubusercontent.com/614929/186225618-dca5108d-8de6-40fc-ab0a-c241667305b3.jpg)

We need to make sure we ask the user precisely which ways are away from the main road (so cycle way and sidewalk or just one of them) and we need to make this clear in the tags.

And I'm currently not sure we can do this - especially with counter flow cycle tracks etc. this gets messy pretty quickly.

**Comment by RubenKelevra** [comment]
One solution would be to stick to the [verge tag](https://wiki.openstreetmap.org/wiki/Key:verge) and do the following:

## Scenario I

![A85_east_of_Crianlarich_-_geograph org uk_-_1542387](https://user-images.githubusercontent.com/614929/186140645-c6b98d08-f7f1-4c32-8b3a-d58360d40272.jpg)

```yaml
verge:both=yes
verge:both:surface=compacted
sidewalk:both=no
```

## Scenario II

![Oak_Park_Boulevard](https://user-images.githubusercontent.com/614929/186140890-7c6dc81e-340a-4c6b-9c56-f10cb013d00e.jpg)

```yaml
verge:both=yes
verge:both:surface=grass
verge:both:width=4m
sidewalk:both=yes
sidewalk:both:surface=concrete
sidewalk:both:width=1m
```

or 


```yaml
verge:both=yes
verge:both:surface=grass
verge:both:width=4m
sidewalk:both=seperate
```

with a separate way for the sidewalks

## Scenario III

![Massachusetts-devils_strip](https://user-images.githubusercontent.com/614929/186223460-96cd6aa0-41be-4166-85a1-759dd46d17b0.JPG)

```yaml
verge:both=yes
verge:both:surface=grass
verge:both:width=1m
sidewalk:left=yes
sidewalk:left:surface=concrete
sidewalk:left:width=1m
sidewalk:right=no
```

or 

```yaml
verge:both=yes
verge:both:surface=grass
verge:both:width=1m
sidewalk:left=seperate
sidewalk:right=no
```


with a separate way for the sidewalk

**Reply by RubenKelevra** [reply]
But main issue here is: If there's a cycle track, how do we make clear that the verge tag applies to it too or not?

**Reply by matkoniecz** [reply]
Are things mentioned here established tags? If not then tagging mailing list would likely be a better place to invent and discuss them.

**Reply by mnalis** [reply]
> Are things mentioned here established tags?

Not really... e.g. [verge:left](https://wiki.openstreetmap.org/wiki/Key:verge:left) has **8** uses worldwide according to taginfo. It is mentioned at top-wiki for [verge=*](https://wiki.openstreetmap.org/wiki/Key:verge) though, although that wiki page seems confusing and looks problematic to me (as noted by @RubenKelevra in this discussion too).

Especially for StreetComplete I do not think such quest would be is useful. 

If someone want to micromap it correctly, by all means feel free to do so - e.g. for **_scenario 3_** mapping road as e.g. `highway=residential,` followed with `landuse=grass` area (speckled with `natural=tree` nodes, or `natural=tree_row` way), followed by `highway=footway`.

Additional advantage of such micromapping is that it would **actually be _rendered_ and _understood_ automatically by routers**, instead of **hypothetical** "well *some* routers *might* pick up the `verge` tags *if* they *somehow* gets popular *sometime* in the future" - which seems unlikely looking at the wiki page, IMHO)

As noted, the issue is complex, and adding simple tags on highways is unlikely to ever be able to solve the problem, even if new tag was vastly better defined than `verge=*` is currently. Mapping as separate ways in IMHO only real way to represent reality in way useful for correct routing (and rendering) - especially as there are different types of verges (e.g. inclined ones would be useless for bicycle routing!), and it would be hugely problematic in SC format to explain all nuances adequately to the users.

In Real Life Situations where it was warranted to allow for **_"much safer routes more easily for bicycles and foot traffic"_** which is the main reason for this suggested quest (e.g. like in situation1 when such a road is motor vehicle only or highly dangerous, and there is short piece of verge as only connection between otherwise unconnected footways/cycleways) I've mapped such cycleable/walkable verges as separate `highway=path`+`surface=grass`+`informal=yes`+`bicycle=yes` manually, so bicycle/pedestrian routing works.

**Reply by RubenKelevra** [reply]
> Are things mentioned here established tags? If not then tagging mailing list would likely be a better place to invent and discuss them.

Not sure why you want to move the discussion - the point was to find out if there are any concurrent tags I'm not aware of and discuss what we would need before we write a proposal which gets pushed to the tagging mailing list.

**Reply by matkoniecz** [reply]
> why you want to move the discussion

Because discussion on GitHub, of a specific project, is not a good place to decide on tagging affecting general osm community

Before adapting (which is not promised) it would need to be consulted anyway.

It is not a good practice to invent a new tagging on issue tracker and start pushing it without consulting. Especially in editors which hide tags from mappers.

In addition tagging discussion tend to generate large discussion volume. And are heavily fragmented already.

It may be tricky, as discussing use of existing tagging is on topic here. But it is definitely inventing a new tagging scheme.
